scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/example/nnframes/xgboost/README.md

```diff
@@ -0,0 +1,64 @@
+# XGBoostClassifier Train Example
+
+## Environment
+- spark 2.4.6
+- BigDL 2.0 
+## Data Prepare
+
+### Build BigDL/scala
+run :
+```
+bash /path/to/BigDL/scala/make-dist.sh
+```
+
+You will get jars include `bigdl-dllib-spark_2.4.6-0.14.0-SNAPSHOT-jar-with-dependencies.jar`
+
+### Get iris.csv
+
+run python command:
```

Comment:
Can we ask user to download the iris.csv file, use it as data source 
 directly and add the python processing code to our example code?

Reply:
It seems that we can use iris.data directly.